src/main/java/com/zhongan/devpilot/integrations/llms/LlmProvider.java

```diff
@@ -47,7 +47,8 @@ public void onEvent(@NotNull EventSource eventSource, @Nullable String id, @Null
                     return;
                 }
 
-                var response = JsonUtils.fromJson(data, DevPilotSuccessStreamingResponse.class);
+                // var response = JsonUtils.fromJson(data, DevPilotSuccessStreamingResponse.class);
```

Comment:
I didn't originally intend to modify this, but the existing code is causing errors. Does it not produce any errors when you run the code ? 

`Unrecognized field "system_fingerprint" (class com.zhongan.devpilot.integrations.llms.entity.DevPilotSuccessStreamingResponse), not marked as ignorable`

there's a property named `system_fingerprint `in `data`, but `DevPilotSuccessStreamingResponse `has no such property. how about add a `system_fingerprint ` property to class `DevPilotSuccessStreamingResponse`?

Reply:
Maybe you can use such "@JsonIgnoreProperties(ignoreUnknown = true)" to ignore unused field

Reply:
ok, I have already modified it according to your suggestion and pushed.